lethal-squirrels/Bangazon-Site
# Matt Ben Ticket#2 Complete Order w/ shoppingcart bug fix

## REFERENCE
The bug where total items wasn't right is fixed
Test Functionality
	* Add multiple of one item
	* Make sure the total items add up.
REFERENCE

Ticket: #2

DESCRIPTION

User can complete an order. Added a viewmodel/method to show user a page where they accept a payment type. When they select one and click complete order, accessed a post method which edits the order and line items and saves changes. Then it redirects to a page where they can review their order details.

STEPS TO TEST

Clone this branch to a local repo
In the package manager, run update-database.
Test functionality
Add products to an order
Complete the order
Check for bugs
